app/package.json

```diff
@@ -11,8 +11,8 @@
     "predev": "yarn run build:schema && yarn run build:relay",
     "build": "yarn run build:server && yarn run build:schema && yarn run build:relay && yarn run build:next",
     "build:server": "tsc --project tsconfig.server.json",
-    "build:schema": "postgraphile -X -c postgres://localhost/cif -s cif --export-schema-graphql server/schema.graphql --export-schema-json server/schema.json --classic-ids --sort-export && echo >> server/schema.json",
-    "build:relay": "relay-compiler --verbose",
+    "build:schema": "postgraphile -X -c postgres://localhost/cif -s cif --export-schema-graphql schema/schema.graphql --export-schema-json schema/schema.json --classic-ids --sort-export && echo >> schema/schema.json",
+    "build:relay": "mkdir -p __generated__ && relay-compiler",
```

Comment:
the new `relay-compiler` doesn't create the artifacts folder by default...but it's FAST